lib/pivotal-tracker/api_service.rb

```diff
@@ -0,0 +1,37 @@
+class PivotalTracker::ApiService
+  def self.find(collection, id)
```

Comment:
Should this class be all the API related stuff currently at the top of pivotal_tracker.rb? i.e. HTTParty, base_uri, PermissionDenied error